clients/src/main/resources/common/message/UpdateMetadataRequest.json

```
@@ -33,6 +33,12 @@
   "validVersions": "0-7",
   "flexibleVersions": "7+",
   "fields": [
+    { "name": "ClusterId", "type": "string", "versions": "7+", "nullableVersions": "7+", "default": "null",
+      "taggedVersions": "7+", "tag": 0, "ignorable": true,
```

1. I'd suggest we rename this to OriginClusterId.
2. I'd suggest that the OriginClusterId and RoutingClusterId to be non-ignorable, and thus cannot be silently discarded if somehow they have a meaningful value and are being serialized to an older message version.

---

Will do.  (I forgot to include this in the latest push, but it will be in the next one.)